server/Startup.cs

```diff
@@ -17,18 +18,20 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+            services.AddScoped<ValidateModelStateAttribute>();
```

Comment:
It's supposed to work for any action with [FromBody] arguments, so in our case all possible POST actions. As for the return types, I'm not sure if it will work with anything else other than IActionResult, but it's better to change all actions to return it explicitly.